app/websocket/app/main/model_manager.py

```diff
@@ -0,0 +1,188 @@
+import syft as sy
+
+import pickle
+import os
+
+from .persistence.models import db, SyftModel
+from sqlalchemy.exc import SQLAlchemyError, IntegrityError
+
+
+model_cache = dict()
+
+
+def cache_get_model(model_id: str):
+    """Checks the cache for a model. If model not found, returns None.
+    
+    Args:
+        model_id (str): Unique id representing the model. 
+
+    Returns:
+        A model encoded in ("ISO-8859-1"), else returns None.
```

Comment:
We actually support multiple encodings, ISO is just the one we're using as the default encoding. Can you remove this from the docstring?

I would write this as: `A model encoded, else returns None.`